contracts/Conference.sol

```diff
@@ -252,6 +252,25 @@ contract Conference is Destructible, GroupAdmin {
         }
     }
 
+    function finalize(uint[] _maps) external onlyAdmin onlyActive {
```

Comment:
Can you add comment above the function for description?

Reply:
Done